.github/workflows/deploy.yml

```
@@ -0,0 +1,110 @@
+        type: choice
+        options:
+        - development
+        - test
+        - staging
+        - production
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)